### PR TITLE
Artik710

### PR DESCRIPTION
**Pull request by shaunmulligan**
@craig-mulligan @imrehg can we get a quick review for this so it can go to prod.

### CODE REVIEW

File: shared/getDeviceOnDash/artik710.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+into the board, **hint:** The SD card slot can be found on the underside of the board near the Audio Jack.
+
+__Note:__ This resinOS will completely write over the existing eMMC.
+
+<!-- <img src="/img/artik5/artik5-dev-kit.png" width="80%"> -->
+If you selected wifi as your connectivity type, remember to attach the wifi antennas before booting the board.
+
+We can now insert the 5VDC power cable and flip the power switch labelled `SW700`, to the `on` position. 
+We should now have some glowing LEDs on the underside of the board indicating a sign of life. Next we need to press and hold the `SW400 POWER` push button for 1 or 2 seconds, 
+this starts the boot from the {{ $device.bootMedia }}. You should see a the red `POWER ON` LED light up when it starts booting.
```
Comment (craig-mulligan) [inline shared/getDeviceOnDash/artik710.md]: You should see **a** the red `POWER ON` -> You should see the red `POWER ON`

File: shared/getDeviceOnDash/artik710.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Your {{ $device.name }} will now flash resinOS onto the internal eMMC so that you can remove the {{ $device.bootMedia }}. This will take 2 or 3 minutes, 
+so time for more tea!! Once it has finished it will shut itself down and you will see the device on the dashboard in a `Post-Provisioning` state. 
+At this point you need to:
+
+1. Flip `SW700` back to `off` position.
+2. Remove the {{ $device.bootMedia }}.
+3. Set `SW402` dip switch to position `1:off`, `2:off`, `3:off`, `4:off`.
+
+After all of that we flip the `SW700` back `on` and once again hold down the `SW400 POWER` button for a second or so. If all goes according to plan we should
```
Comment (craig-mulligan) [inline shared/getDeviceOnDash/artik710.md]: After all of that**,**

File: shared/selectNetworkConfig/artik710.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Before your resinOS download begins, a prompt will appear asking you to specify which version of resinOS you want to run and how your device will connect to the internet. 
+Make sure to select `Resin OS 2.0.2-beta.2` from the dropdown menu.
+
+Currently there are two connectivity options:
+ * Ethernet cable, this option requires *NO* configuration as is the default.
+ * Wifi, in which case you must specify the [network name or`SSID`][ssid-link] and `passphrase` for the network your device will connect to.
+
+Once you have selected your connectivity option, click the `Download Device OS` button to get the resin.io operating system image configured for your application and network.
```
Comment (craig-mulligan) [inline shared/selectNetworkConfig/artik710.md]: It was called `Download resinOS` in the previous section

File: shared/getDeviceOnDash/artik710.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+We can now insert the 5VDC power cable and flip the power switch labelled `SW700`, to the `on` position. 
+We should now have some glowing LEDs on the underside of the board indicating a sign of life. Next we need to press and hold the `SW400 POWER` push button for 1 or 2 seconds, 
+this starts the boot from the {{ $device.bootMedia }}. You should see a the red `POWER ON` LED light up when it starts booting.
+
+After a few seconds you should see your {{ $device.name }} appear on the resin.io dashboard in a configuring state. 
+If it still hasn't appeared after 1 or 2 minutes, double check your ethernet connection or that you entered the right wifi credentials at download time. 
+If you still can't get it online, contact us on support.
+
+Your {{ $device.name }} will now flash resinOS onto the internal eMMC so that you can remove the {{ $device.bootMedia }}. This will take 2 or 3 minutes, 
+so time for more tea!! Once it has finished it will shut itself down and you will see the device on the dashboard in a `Post-Provisioning` state.
```
Comment (craig-mulligan) [inline shared/getDeviceOnDash/artik710.md]: I think one exclamation will suffice :)

**Comment by imrehg** [comment]
LGTM

File: shared/selectNetworkConfig/artik710.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+Before your resinOS download begins, a prompt will appear asking you to specify which version of resinOS you want to run and how your device will connect to the internet. 
+Make sure to select `Resin OS 2.0.2-beta.2` from the dropdown menu.
```
Comment (craig-mulligan) [inline shared/selectNetworkConfig/artik710.md]: Should we explain why we don't recommend the recommended?

File: shared/getDeviceOnDash/artik710.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+<!-- <img src="/img/artik5/artik5-dev-kit.png" width="80%"> -->
+If you selected wifi as your connectivity type, remember to attach the wifi antennas before booting the board.
+
+We can now insert the 5VDC power cable and flip the power switch labelled `SW700`, to the `on` position. 
+We should now have some glowing LEDs on the underside of the board indicating a sign of life. Next we need to press and hold the `SW400 POWER` push button for 1 or 2 seconds, 
+this starts the boot from the {{ $device.bootMedia }}. You should see a the red `POWER ON` LED light up when it starts booting.
+
+After a few seconds you should see your {{ $device.name }} appear on the resin.io dashboard in a configuring state. 
+If it still hasn't appeared after 1 or 2 minutes, double check your ethernet connection or that you entered the right wifi credentials at download time. 
+If you still can't get it online, contact us on support.
```
Comment (craig-mulligan) [inline shared/getDeviceOnDash/artik710.md]: We should have a doc on our support channels so we can link to it in cases like this yea?

File: shared/selectNetworkConfig/artik710.md (hunk 5 above)
Comment (shaunmulligan) [inline shared/selectNetworkConfig/artik710.md]: @craig-mulligan there is an issue, where there is no recommended, its just a blank drop down, this image will become the recommended when that bug is fixed

File: shared/getDeviceOnDash/artik710.md (hunk 4 above)
Comment (shaunmulligan) [inline shared/getDeviceOnDash/artik710.md]: no ways!!!!

File: shared/getDeviceOnDash/artik710.md (hunk 6 above)
Comment (shaunmulligan) [inline shared/getDeviceOnDash/artik710.md]: I think we already do, its just not in the menu: https://docs.resin.io/support/

**Comment by shaunmulligan** [comment]
fixed the remaining comments in https://github.com/resin-io/docs/pull/351

**Pull request merged**